piety-promoted-v1/modernized/15-chapter-14.adoc

```diff
@@ -165,7 +172,10 @@ for he never deserved it at my hands.`'
 
 Then she also desired him to write a letter for her to her husband, as follows:
 
-'`Dear and loving husband, William Berry.
+[.embedded-content-document.letter]
+--
+
+Dear and loving husband, William Berry.
```

Comment:
I would mark this first line as a salutation. 👍🏻

Reply:
So I made them offset and they are spaced pretty far apart, but it looked better than embedding them because they were not distinguished from each other enough with that...it looked like one big block of text. Idk if there is any other way to make them stand out a little but appear slightly closer together?